boards/components/src/hmac.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! Usage
 //! -----
-//! ```rust
+//! ```ignore
```

Comment:
I think it should be `rust,ignore`

Reply:
it should be just rust, that was a leftover from earlier debugging